modules/nf-core/muse/sump/main.nf

```diff
@@ -0,0 +1,51 @@
+process MUSE_SUMP {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'oras://community.wave.seqera.io/library/muse:2.1.2--f6ec9e78771509ff':
+        'community.wave.seqera.io/library/muse:2.1.2--e8279641c6ef8c63' }"
+
+    input:
+    tuple val(meta), path(muse_call_txt)
+    tuple val(meta2), path(ref_vcf), path(ref_vcf_tbi)
+
+    output:
+    tuple val(meta), path("*.vcf"), emit: vcf
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: '' // hands -G for WGS data and -E for WES data
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def VERSION = '2.1.2' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+    """
+    MuSE \\
+        sump \\
+        $args \\
+        -I $muse_call_txt \\
+        -O ${prefix}.vcf \\
```

Comment:
Just a thought, how big is the vcf here? Does it want to be zipped?

Reply:
How big is the output vcf going to be?

Reply:
depends on the input (but could potentially be large) - i added bgzip now maybe you can check if it makes sense the way I did it